watershed-rails/db/seeds/development/005_tasks.rb

```diff
@@ -20,7 +20,7 @@
 		description: tasks.sample,
 		assignee_id: User.first,
 		assigner_id: User.last,
-		site_id: Site.first,
+		mini_site_id: MiniSite.first,
```

Comment:
also, I think we might move away from the Site/Minisite terminology, so stay tuned for that